src/plugins/kibana_react/public/table_list_view/table_list_view.tsx

```diff
@@ -440,12 +441,18 @@ class TableListView extends React.Component<TableListViewProps, TableListViewSta
       },
     };
 
+    const actionsColumnName = i18n.translate(
+      'kibana-react.tableListView.listing.table.actionTitle',
+      {
+        defaultMessage: 'Actions',
+      }
+    );
+
     const columns = this.props.tableColumns.slice();
     if (this.props.editItem) {
       columns.push({
-        name: i18n.translate('kibana-react.tableListView.listing.table.actionTitle', {
-          defaultMessage: 'Actions',
-        }),
+        name: actionsColumnName,
+        field: actionsColumnName,
```

Comment:
I prefer to fix that issue on **Elastic UI** side. `field` param is not required for `ActionsColumn` (see https://elastic.github.io/eui/#/tabular-content/in-memory-tables). From my point of view `ActionsColumn`'s  should be ignored by default for sorting

Reply:
@chandlerprall any ideas? Should we move that fix to **Elastic UI** side?

Reply:
@chandlerprall any ideas? Should we move that fix to Elastic UI side?

Reply:
Agreed, please create an issue in https://github.com/elastic/eui and we'll take a look.